force:
  proportional:
    x: 0.7 
    y: 0.7
    z: 4
  differential:
    x: 0.4
    y: 0.4
    z: 2.3

yawcontrol:
  proportional: 0.030
  differential: 0.005

rollcontrol:
  proportional: 0.5
  differential: 0.05

pitchcontrol:
  proportional: 0.5
  differential: 0.05

rate_damping:
  roll: -0.04
  pitch: 0.04

motorconst:
  const1: 0.35
  const2: 0.3832

rollrccoef:
  coef: 0.4

pitchrccoef:
  coef: 0.4

rollconst:
  const: -0.050

pitchconst:
  const: 0.008

throttlecoef:
  coef1: 0.52
  coef2: 0.52

yawcoef:
  coef1: 0.0001
  coef2: 0.0001
